### PR TITLE
docs: fix alignment of line number with code examples

### PR DESCRIPTION
**Pull request by Tanujkanti4441**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)
just added a line height property to syntax-highlighter.scss file for making the alignment of line numbers with the code in examples

**before.**
Screenshot 2022-12-16 000348

**after this PR**

Screenshot 2022-12-16 001930

### CODE REVIEW

**Comment by amareshsm** [comment]
Let's wait for this PR https://github.com/eslint/eslint/pull/16669. It includes some changes in styling also.

File: docs/src/assets/scss/syntax-highlighter.scss [hunk 1]
```diff
@@ -118,6 +118,7 @@ pre[class*="language-"] {
     font-variant-numeric: tabular-nums;
     text-align: right;
     width: 1.2em;
+    line-height: 0;
```
Comment (chenxsan) [inline docs/src/assets/scss/syntax-highlighter.scss:121]: Mind sharing which browser and its version as well as the OS to reproduce the problem?

Comment (Tanujkanti4441) [reply docs/src/assets/scss/syntax-highlighter.scss:121]: I am seeing this problem in
 **Browser** - Firefox
**Version** - 108.0.1 (latest)
**Operating system** - Windows 10 Pro (64-bit)

I am not getting this problem in **Chrome** browser but this fix (PR) is not affecting that

Comment (chenxsan) [reply docs/src/assets/scss/syntax-highlighter.scss:121]: Thanks, I can reproduce it in Firefox.

Seems the `line-height` for `code` has no effect under firefox, I'll see why it happens.

Actually I can find the problem even before my pull request (screenshots from https://eslint.org/docs/latest/rules/array-callback-return under firefox):





My pull request happened to expose it :)

Comment (chenxsan) [reply docs/src/assets/scss/syntax-highlighter.scss:121]: So I compared the css rules before to those after my pull request and found there was one rule missing from my pr. Applying that rule to `.line-numbers-wrapper` seems to fix the problem under firefox:

```
.line-numbers-wrapper {
  font-variant-ligatures: none;
}
```
Could you try it and verify if it works for you? I reckon it doesn't make sense to set `line-height` to `0`, though it did work in this case under firefox.

Comment (Tanujkanti4441) [reply docs/src/assets/scss/syntax-highlighter.scss:121]: Yes, it is working for me,
ok then, i am closing this PR issue is all yours

Comment (chenxsan) [reply docs/src/assets/scss/syntax-highlighter.scss:121]: Thanks, I've updated that pull request to fix this issue.

**Pull request closed without merging**